platform-includes/distributed-tracing/custom-instrumentation/server/javascript.mdx

```diff
@@ -0,0 +1,92 @@
+Trace propagation is set up automatically in this SDK. By default, outgoing HTTP requests will automatically be instrumented for you.
+For other cases where you may want to continue traces (for example, in a job queue or when working with websockets), you can manually extract and inject tracing information.
+
+<Expandable title="What is Distributed Tracing?">
+  <PlatformContent includePath="distributed-tracing/explanation" />
+</Expandable>
+
+## Manually Extracting and Injecting Distributed Tracing Information
+
+You can also manually extract and inject tracing data into your application. For this, you must:
+
+- Extract and store incoming tracing information from incoming request headers or similar.
+- Inject tracing information to any outgoing requests.
+
+To learn more about distributed tracing, see our <PlatformLink to="/tracing/trace-propagation/">Distributed Tracing</PlatformLink> docs.
+
+### Extracting Incoming Tracing Information
+
+You must extract and store incoming tracing information in memory for later use. Sentry provides the `continueTrace()` function to help you with this. Incoming tracing information can come from different places:
+
+- In a web environment, it's sent with HTTP headers, for example, by another Sentry SDK used in your frontend project.
+- In a job queue, it can be retrieved from meta or header variables.
+- You also can pick up tracing information from environment variables.
+
+Here's an example of how to extract and store incoming tracing information using `continueTrace()`:
+
+```javascript
+const http = require("http");
+
+http.createServer((request, response) => {
+  const sentryTraceHeaders = request.headers["sentry-trace"];
+  const sentryTrace = Array.isArray(sentryTraceHeaders)
```

Comment:
m: I don't think joining multiple sentry-trace headers by `,` is [valid](https://github.com/getsentry/sentry-javascript/blob/1d797ea3ee4653ccea20cb217195a79766689491/packages/core/src/utils-hoist/tracing.ts/#L8-L14). I guess this could happen for _reasons_ but is it worth pointing this out in the docs? If at all, we should probably just take one? Or any chance this got mixed up with `baggage`? baggage arrays could happen I guess if multiple instrumentation libraries add baggage headers 🤔